ntrans_gui.py

```diff
@@ -0,0 +1,293 @@
+        self.root.resizable(False, False)
+        self.root.title("N-Trans")
+
+        self.mainframe = ttk.Frame(self.root)
+        self.mainframe.pack(fill="both", expand=True)
+
+        # Header
+        header = ttk.Label(
+            self.mainframe,
+            text="N-Trans Dictionary Settings",
+            font=("TkDefaultFont", 18),
+        )
+
+        # Get directory path to save csv file
+        # TODO: Add default save path depending on OS
+        get_savepath_button = ttk.Button(
+            self.mainframe, command=self.get_save_file_path, text="Browse..."
+        )
+
+        self.filepath = tkinter.StringVar()
+
+        filepath_label = ttk.Label(self.mainframe, text="Save N-Trans dictionary to")
+
+        show_filepath = ttk.Entry(self.mainframe, textvariable=self.filepath, width=25)
+
+        # What N-Grams to include in csv file.
+        ngram_check_label = ttk.Label(self.mainframe, text="Choose N-grams to include")
+
+        self.deselected = 0
+        self.selected = 1
+
+        self.checked_select_all = tkinter.IntVar(value=self.selected)
+        self.checked_2_gram = tkinter.IntVar(value=self.selected)
+        self.checked_3_gram = tkinter.IntVar(value=self.selected)
+        self.checked_4_gram = tkinter.IntVar(value=self.selected)
+        self.checked_5_gram = tkinter.IntVar(value=self.selected)
+        self.checked_6_gram = tkinter.IntVar(value=self.selected)
+
+        select_all_check = ttk.Checkbutton(
+            self.mainframe,
+            text="Select all",
+            variable=self.checked_select_all,
+            command=self.select_all_ngrams,
+        )  # TODO: Add method that selects/deselects all and changes the text to reflect state.
+
+        two_gram_check = ttk.Checkbutton(
+            self.mainframe,
+            text="2-grams",
+            variable=self.checked_2_gram,
+            command=self.update_ngram_checkbox(2),
+        )
+
+        three_gram_check = ttk.Checkbutton(
+            self.mainframe, text="3-grams", variable=self.checked_3_gram
+        )
+
+        four_gram_check = ttk.Checkbutton(
+            self.mainframe, text="4-grams", variable=self.checked_4_gram
+        )
+
+        five_gram_check = ttk.Checkbutton(
+            self.mainframe, text="5-grams", variable=self.checked_5_gram
+        )
+
+        six_gram_check = ttk.Checkbutton(
+            self.mainframe, text="6-grams", variable=self.checked_6_gram
+        )
+
+        # What language to translate into
+        target_language_label = ttk.Label(self.mainframe, text="Target language")
+
+        self.target_language_var = tkinter.StringVar()
+
+        target_language = ttk.Combobox(
+            self.mainframe, state="readonly", textvariable=self.target_language_var
+        )
+
+        target_language["values"] = (
+            "German",
+            "Spanish",
+            "French",
+            "Swedish",
+            "Italian",
+        )
+
+        target_language.current()
```

Comment:
Is this needed? It seems to just return the current value, but the returned value is not used for anything.

Reply:
The thought was that this would make sure the Combobox was empty when starting the program, but it defaults to empty, so I removed it. Thanks